style.css

```diff
@@ -0,0 +1,6 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  height: auto;
+}
```

Comment:
Great work adding a style sheet to your page. But I will suggest you also add background color and color to your text. This will bring out the beauty of your page and make it more vibrant and attractive.